generators/Exercises/TwelveDays.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Generators.Input;
+using Generators.Output;
+using System.Linq;
+using Humanizer;
+
+namespace Generators.Exercises
+{
+    public class TwelveDays : GeneratorExercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            canonicalData.Exercise = "twelve-days-song";
```

Comment:
Why the changed exercise name?

Reply:
I understand, but it is in fact the other way around: we should be following the canonical data whenever possible.